charts/scalardb/values.yaml

```diff
@@ -123,6 +123,22 @@ scalardb:
     # -- The log level of Scalar DB
     dbLogLevel: INFO
 
+  useCustomizedConfiguration:
+    # -- Use user customized database.properties. You need to create ConfigMap includes database.properties.
+    enabled: false
+    # -- ConfigMap name that includes database.properties.
+    configMapName: "scalardb-customized-config"
+    # -- Use Secret to pass the credentials as environment variable.
+    useSecret: false
+    # -- Secret name that includes credentials.
+    secretName: "scalardb-customized-secret"
+    # -- Array of hash that includes environment variable name and secret key name.
+    secretKeys:
+      - environmentVariableName: SCALAR_DB_USERNAME
+        secretKeyName: db-username
+      - environmentVariableName: SCALAR_DB_PASSWORD
+        secretKeyName: db-password
```

Comment:
You can consider using envFrom. In this case, the key name of the Secret resource becomes the environment variable name, and this setting becomes unnecessary.

- https://kubernetes.io/docs/concepts/configuration/secret/#use-case-as-container-environment-variables

I think there is no need to separate Secret key names from environment variable names.

Reply:
I didn't know the `envFrom`. Thank you!

I updated the deployment.yaml to use `envFrom`.
Also, I removed this unnecessary settings from values.yaml.